package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "got",
-	"version": "11.6.1",
+	"version": "11.6.0",
```

```suggestion
	"version": "11.6.1",
```

i have no idea why it reverted to .0